api/modules/automations.js

```diff
@@ -30,20 +30,43 @@ const automations = module.exports = (() => {
     }
 
     const getOrganizationRepos = async (params) => {
-        const organizations = await getUserOrganizations({
-            auth_key: params.auth_key
-        })
-        const isOrganization = organizations[0].name != params.organizationName
         
+        const contributor = await db.models.Contributor.findOne({
+            where: {
+                github_id: params.accountId 
+            }
+        })
+
+        let organizations
+        let accountName
+        let isOrganization = false
+        if (contributor) {
+            const contributorSplit = contributor.github_handle.split(/\//)
+            accountName = contributorSplit[3]
+        } else {
+            organizations = await getUserOrganizations({
+                auth_key: params.auth_key
+            })
+        }
+
+        if (organizations) {
+            isOrganization = true
+            organizations.map((org, i) => {
+                if (org.id == params.accountId) {
+                    accountName = org.name
+                }
+            })
+        }
+        console.log(accountName, isOrganization)
```

Comment:
Always try to think about solutions without the use o `var` and `let` and use `const` instead
To achieve that in this logic you can do
```
const organizations = []
const githubAccount = {}
if (contributor) {
    githubAccount.handler = contributor.github_handle.split(/\//)[3]
 } else {
     organizations.push(
            ...await getUserOrganizations({
                          auth_key: params.auth_key
             })
       )
 }

if (organizations.length) {
            organizations.map((org, i) => {
                if (org.id == params.accountId) {
                    githubAccount.handle = org.name
                }
            })
}


const repos = await github.fetchRepos({
            auth_key: params.auth_key,
            organizationName: params.organizationName,
            accountName: githubAccount.handle,
            githubPageNumber: params.githubPageNumber,
            isOrganization: organizations.length
})
```